speech/snippets/multichannel_test.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package snippets
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestMultichannel(t *testing.T) {
+	testutil.SystemTest(t)
+
+	var buf bytes.Buffer
+	err := transcribeMultichannel(&buf, "../testdata/commercial_stereo.wav")
+	if err != nil {
```

Comment:
Prefer inline declarations if none of the variables are needed outside the if statement. Helps to avoid forgetting to check the error or accidentally shadowing a variable.

```suggestion
	if err := transcribeMultichannel(&buf, "../testdata/commercial_stereo.wav"); err != nil {
```

Reply:
got it